packages/css/src/components/password-input/password-input.scss

```
@@ -0,0 +1,59 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+@import "../../common/text-rendering";
+
+@mixin reset {
+  -webkit-appearance: none; // Reset appearance for Safari < 15.4
+  appearance: none; // Reset native appearance, this causes issues on iOS and Android devices
+  border: 0;
+  border-radius: 0; // Reset rounded borders on iOS devices
+  box-sizing: border-box;
+  margin-block: 0;
+}
+
+.ams-password-input {
+  background-color: var(--ams-password-input-background-color);
+  box-shadow: var(--ams-password-input-box-shadow);
+  color: var(--ams-password-input-color);
+  font-family: var(--ams-password-input-font-family);
+  font-size: var(--ams-password-input-font-size);
+  font-weight: var(--ams-password-input-font-weight);
+  inline-size: 100%;
+  line-height: var(--ams-password-input-line-height);
+  outline-offset: var(--ams-password-input-outline-offset);
+  padding-block: var(--ams-password-input-padding-block);
+  padding-inline: var(--ams-password-input-padding-inline);
+  touch-action: manipulation;
+
+  @include text-rendering;
+  @include reset;
+
+  &:hover {
+    box-shadow: var(--ams-password-input-hover-box-shadow);
+  }
+}
+
+.ams-password-input::placeholder {
+  color: var(--ams-text-input-placeholder-color);
```

```suggestion
  color: var(--ams-password-input-placeholder-color);
```

The token is missing as well.

---

Fixed and added in a commit.